### ISSUE TITLE
Add shrinkwrap back into nsp check task

### ISSUE THREAD

**Issue by DonutEspresso** [restify/node-restify#1492]
- [x] Used appropriate template for the issue type
- [x] Searched both open and closed issues for duplicates of this issue
- [x] Title adequately and _concisely_ reflects the feature or the bug

A recent release of nsp caused nsp's internal dev dependencies to be included as part of the nsp check. This is a bug on nsp's part:
nodesecurity/nsp#185

We should revert #1482 once a new nsp version is released.

**Comment by hekike** [comment]
nsp finds vulnerabilities in the dependency tree without shrinkwrap if we have a `package-lock.json` file. Shouldn't we add `package-lock.json` to restify instead of reverting $1482?

With `package-lock.json` file:

```
$ make nsp
(+) 1 vulnerabilities found
┌───────────────┬─────────────────────────────────────────────────────────────────┐
│               │ Regular Expression Denial of Service                            │
├───────────────┼─────────────────────────────────────────────────────────────────┤
│ Name          │ debug                                                           │
├───────────────┼─────────────────────────────────────────────────────────────────┤
│ CVSS          │ 3.7 (Low)                                                       │
├───────────────┼─────────────────────────────────────────────────────────────────┤
│ Installed     │ 2.6.8                                                           │
├───────────────┼─────────────────────────────────────────────────────────────────┤
│ Vulnerable    │ <= 2.6.8 || >= 3.0.0 <= 3.0.1                                   │
├───────────────┼─────────────────────────────────────────────────────────────────┤
│ Patched       │ >= 2.6.9 < 3.0.0 || >= 3.1.0                                    │
├───────────────┼─────────────────────────────────────────────────────────────────┤
│ Path          │ restify@6.0.1 > spdy@3.4.7 > debug@2.6.8                        │
├───────────────┼─────────────────────────────────────────────────────────────────┤
│ More Info     │ https://nodesecurity.io/advisories/534                          │
└───────────────┴─────────────────────────────────────────────────────────────────┘
```

**Comment by hekike** [comment]
Related to: https://github.com/restify/node-restify/pull/1477
